source/content/guides/secure-development/05-two-factor-authentication.md

```diff
@@ -0,0 +1,255 @@
+
+1. Enter the six-digit TFA code on your mobile app for your specific site to complete the setup. You will then be prompted to confirm a trusted browser (which is optional and will skip TFA on that browser in the future), and to write down TFA recovery codes (best practice).
+
+1. Log in to your Drupal site by using the TOTP mobile app to generate a six digit code.
+
+  ![TFA setup](../../../images/tfa-drupal-login.png)
+
+</Tab>
+
+</TabList>
+
+
+
+## Organization TFA
+
+<TabList>
+
+<Tab title="WordPress" id="wp-org" active={true}>
+
+There are many different organization-wide [WordPress plugins for single sign on](https://wordpress.org/plugins/tags/single-sign-on) that can provide TFA capabilities. One of the service options we use internally at Pantheon is OneLogin, which has the [OneLogin SAML SSO](https://wordpress.org/plugins/onelogin-saml-sso/) plugin.
+
+#### OneLogin Instructions
+
+1. Sign up and create a [OneLogin account](https://www.onelogin.com/) for your organization.
+
+1. Install the WordPress SAML 2.0 app connector as part of the OneLogin dashboard (you need administrator privileges to install apps). This must be done for each WordPress site that is being managed by OneLogin.
+
+1. Edit the OneLogin WordPress app connector to provide the appropriate default values for the Configuration section. Other sections should already be set up correctly.
+
+  ![TFA OneLogin Config](../../../images/tfa-wp-onelogin-config.png)
+
+1. **(Optional)** Configure the **Authentication Factors** found under Settings for a list of authentication factors you can enable for your different users.
+
+  ![TFA OneLogin Methods](../../../images/tfa-onelogin-tfa-methods.png)
+
+1. Create user accounts in the Users Administration area of OneLogin.
+
+1. Click **New User** and verify that the “Username” and "Email" fields in OneLogin match their WordPress username and email.
+
+  ![TFA OneLogin New User](../../../images/tfa-onelogin-new-user.png)
+
+#### WordPress Instructions
+
+1. Install and activate the [OneLogin SAML SSO](https://wordpress.org/plugins/onelogin-saml-sso/) plugin on your WordPress site.
+
+1. Configure the **Identity Provider Settings** section in the SSO/SAML Settings within the WordPress Admin to provide the appropriate values, which are available in the SSO section of the OneLogin Configuration page.
+
+  ![TFA OneLogin Ident](../../../images/tfa-wp-onelogin-ident.png)
+
+1. Configure the **Options** section (optional) under the SSO/SAML Settings:
+
+   - Select the **Create user if not exists** checkbox if you want users to be auto-created.
+
+   - Select the **Keep Local login** checkout if you still want to use the normal WP login form, otherwise you will always be using OneLogin to authenticate.
+
+   ![TFA OneLogin Options](../../../images/tfa-wp-onelogin-options.png)
+
+1. Configure the **Attribute Mapping** in the SSO/SAML Settings of the WordPress Admin with the values shown below. Values are case-sensitive.
```

Comment:
The values need to be included as text, for accessibility purposes.  I would have done so myself, but wasn't sure if they required any placeholder values.  Also, once the values are included as text, you may not need the image anymore.